python/pyspark/ml/evaluation.py

```diff
@@ -409,9 +410,9 @@ class MulticlassClassificationEvaluator(JavaEvaluator, HasLabelCol, HasPredictio
     """
     metricName = Param(Params._dummy(), "metricName",
                        "metric name in evaluation "
-                       "(f1|accuracy|weightedPrecision|weightedRecall|weightedTruePositiveRate|"
-                       "weightedFalsePositiveRate|weightedFMeasure|truePositiveRateByLabel|"
-                       "falsePositiveRateByLabel|precisionByLabel|recallByLabel|fMeasureByLabel|"
+                       "(f1|accuracy|weightedPrecision|weightedRecall|weightedTruePositiveRate| "
```

Comment:
I guess we do not need to add spaces here

Reply:
Seems I have to break the line by adding a white space.
Before:
![image](https://user-images.githubusercontent.com/13592258/75801781-89353380-5d30-11ea-9dda-1e81cf3b016c.png)

After:
![image](https://user-images.githubusercontent.com/13592258/75801816-95b98c00-5d30-11ea-9b29-364ba247ece6.png)